docs: mention locator.nth is zero based (#12750)
microsoft/playwright: docs/src/api/class-framelocator.md

## method: FrameLocator.nth
- returns: <[FrameLocator]>

> Returns locator to the n-th matching frame. It's zero based and starts by that at `0`.

Reviewer: ```suggestion
Returns locator to the n-th matching frame. It's zero based, `nth(0)` selects the first element.
```
